src/modules/feature-modules/admin/pages/organisations/organisations-list.component.ts

```diff
@@ -43,35 +50,19 @@ export class PageOrganisationsListComponent extends CoreComponent implements OnI
 
         this.organisations = organisationUnits.map(organisation => {
 
-          if (organisation.organisationUnits.length === 1) {
-            return {
-              info: {
-                id: organisation.id,
-                name: organisation.name,
-                acronym: organisation.acronym,
-                isActive: organisation.isActive,
-                organisationUnits: [],
-              },
-              showHideStatus: 'hidden',
-              showHideText: null,
-              showHideDescription: null
-            };
-          } else {
-            return {
-              info: {
-                id: organisation.id,
-                name: organisation.name,
-                acronym: organisation.acronym,
-                isActive: organisation.isActive,
-                organisationUnits: organisation.organisationUnits
-              },
-              showHideStatus: 'closed',
-              showHideText: organisation.organisationUnits.length === 0 ? null : `Show ${organisation.organisationUnits.length} units`,
-              showHideDescription: `that belong to the ${organisation.name}`
-            };
-          }
+          return {
+            info: {
+              id: organisation.id,
+              name: organisation.name,
+              acronym: organisation.acronym,
+              isActive: organisation.isActive,
+              organisationUnits: organisation.organisationUnits
+            },
+            unitText: organisation.organisationUnits.length === 0 ? null : `${organisation.organisationUnits.length} units attached`,
+          };
+        }
 
-        });
+        );
```

Comment:
Since the return is not needed anymore, you can make this shorter:

```
(...).map(organisation => ({
    info: { ... },
    unitText: ...
}))
```